boards/reel_board.json

```diff
@@ -0,0 +1,44 @@
+{
+  "build": {
+    "core": "nRF5",
```

Comment:
This field is mostly used with Arduino framework. nRF5 core doesn't support this board, so it seems redundant to me.